src/components/TextInput/styles.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { Inter } from 'next/font/google';
+import COLORS from '../../styles/colors';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export const InputText = styled.input<{ $error: boolean }>`
+  color: ${COLORS.existingInputColor};
+  font-size: 1rem; // 16px
+  font-family: ${inter.style};
+  font-weight: 400;
+  line-height: 19.2px;
+  word-wrap: break-word;
+  padding: 10px 20px 10px 20px;
+  border-radius: 5px;
+  border: 1px solid
+    ${({ $error }) => ($error ? COLORS.errorColor : COLORS.existingInputColor)};
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 10px;
+  color: ${({ $error }) =>
+    $error ? COLORS.errorColor : COLORS.existingInputColor};
+
+  &:focus {
+    outline: none !important;
+    border: 1px solid
+      ${({ $error }) => ($error ? COLORS.errorColor : COLORS.activeGreen)};
+  }
+  &::placeholder {
+    color: ${({ $error }) =>
+      $error ? COLORS.errorColor : COLORS.inputSuggestionColor};
```

Comment:
We may not want the placeholder color to be red (or at least the same red as the value color), since it will get confusing to the user whether or not the text in the input field is there.

If we want to convey to the user that the error was because of no value, we could insert a small text below the input (handled outside of the component), and the red border can also convey that.

Reply:
mmm i get what you're saying! On the figma, even the placeholder text is red, so i'm going to loop in @davidqing6432 for his thoughts!

Reply:
yeah definitely a david type question 🖌️

Reply:
i incorporated david's feedback! see updated screenshots